html/browsers/history/the-history-interface/history_go_to_uri.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<title>history.go()</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<div id="log"></div>
+<script>
+  var t = async_test(undefined, {timeout:5000});
```

Comment:
The name is taken from document.title if omitted, which is fine when it's just one test in a file.